app/javascript/components/signup_form.jsx

```diff
@@ -0,0 +1,75 @@
+// eslint-disable-next-line no-use-before-define
+import React from "react";
+import $ from "jquery";
+
+function SignupForm () {
+  const [name, setName] = React.useState("");
+  const [sname, setSname] = React.useState("");
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [confirmpassword, setConfirm] = React.useState("");
+  const [errorsList, setErrors] = React.useState([]);
+
+  const SendForm = (e) => {
+    e.preventDefault();
+    $.ajax({
+      async: false,
+      type: "POST",
+      url: "/create",
+      data: {
+        first_name: name,
+        second_name: sname,
+        email: email,
+        password: password,
+        password_confirmation: confirmpassword
+      },
+
+      success: function (result) {
+        setErrors(result.errors);
+      },
+      error: function (result) {
+
```

Comment:
You are not doing anything within this block. What kind of a result do you get in this instance and what can you do with it?

Reply:
I'm not too sure as to what I should be doing with the result of the error, as I believe this error will not occur very commonly. For now I've added in an 'alert' that asks the user to 'Please try again'

Reply:
I don't see the code to alert a message, have you pushed it up?

Reply:
Its pushed, not sure why the file isn't updating for you. Ive tagged you in a comment where it is updated